tests/test_config.py

```diff
@@ -125,27 +149,30 @@ def create_app():
     assert len(caplog.records) == 1
     assert "--factory" in caplog.records[0].message
 
-    # App not a no-arguments callable.
+    # App not a no-arguments ASGIApplication.
```

Comment:
I dont get this change, I'd leave it as is

Reply:
Yeah, me too.

Reply:
whoops, find and replace strikes again :zany_face: